jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -100,35 +97,39 @@ pipeline {
                     def CREDENTIAL_ID = 'jenkins-production-dockerhub-credential-readonly'
 
                     echo("Build Cause: ${BUILD_CAUSE}")
-                    withCredentials([usernamePassword(credentialsId: CREDENTIAL_ID, usernameVariable: 'DOCKER_USERNAME', passwordVariable: 'DOCKER_PASSWORD')]) {
-                        def dockerLogin = sh(returnStdout: true, script: "set +x && echo $DOCKER_PASSWORD | docker login --username $DOCKER_USERNAME --password-stdin").trim()
-
-                        if (!env.BUILD_CAUSE.contains('Started by user') && !env.BUILD_CAUSE.contains('Started by timer')) {
-                            def pr_url = "${pr_to_clone_url}".replace(".git", "/pull/${pr_number}")
-                            println("Triggered by GitHub: ${pr_to_clone_url}")
-                            if ("$pr_number" == "Null") {
-                                currentBuild.description = """runner: ${agent_name}<br><a href="${pr_to_clone_url}">Others</a>: ${pr_title}"""
-                            }
-                            else {
-                                currentBuild.description = """runner: ${agent_name}<br><a href="${pr_url}">PR #${pr_number}</a>: ${pr_title}"""
-                            }
+                    if ( ! params.AGENT_LABEL.toLowerCase().contains('windows')) {
+                        withCredentials([usernamePassword(credentialsId: CREDENTIAL_ID, usernameVariable: 'DOCKER_USERNAME', passwordVariable: 'DOCKER_PASSWORD')]) {
+                            def dockerLogin = sh(returnStdout: true, script: "set +x && echo $DOCKER_PASSWORD | docker login --username $DOCKER_USERNAME --password-stdin").trim()
+                        }
+                    }
 
-                            runGradleCheck(
-                                gitRepoUrl: "${pr_from_clone_url}",
-                                gitReference: "${pr_from_sha}"
-                            )
+                    if (!env.BUILD_CAUSE.contains('Started by user') && !env.BUILD_CAUSE.contains('Started by timer')) {
+                        def pr_url = "${pr_to_clone_url}".replace(".git", "/pull/${pr_number}")
+                        println("Triggered by GitHub: ${pr_to_clone_url}")
+                        if ("$pr_number" == "Null") {
```

Comment:
Didn't understand this case?
Can a PR not have a PR number or is this some other trigger?

Reply:
This is just changed due to spacing, the old itself is not changed for months.
It literally just to determine whether if this is triggered by PR then assign number of that PR in description, else dont.